auacm/angular/src/app/app.module.ts

```diff
@@ -0,0 +1,97 @@
+import { BrowserModule } from '@angular/platform-browser';
```

Comment:
For the most part they are organized alphabetically, but they are split up into different components, such as services, components, modules, directives, and pipes. I can change it so they are all bundled together if you want though